2021-09-04 20:09:35,389 INFO    MainThread:392 [wandb_setup.py:_flush():69] setting env: {'entity': 'zumo09', 'project': 'flatland-rl-prova-sweep', 'sweep_id': 'y0xao9ob', 'root_dir': 'D:\\Users\\Zumo\\Documents\\UNIVERSITÀ\\AI\\DL\\Flatland_DDDQN', 'run_id': 'z6ziqb72', 'sweep_param_path': 'D:\\Users\\Zumo\\Documents\\UNIVERSITÀ\\AI\\DL\\Flatland_DDDQN\\wandb\\sweep-y0xao9ob\\config-z6ziqb72.yaml'}
2021-09-04 20:09:35,389 INFO    MainThread:392 [wandb_setup.py:_flush():69] setting login settings: {'api_key': '0f20b9f069a2312cc2b8e92e6f75310697d5fdfc'}
2021-09-04 20:09:35,390 INFO    MainThread:392 [wandb_setup.py:_flush():69] setting login settings: {'api_key': '0f20b9f069a2312cc2b8e92e6f75310697d5fdfc'}
2021-09-04 20:09:35,390 INFO    MainThread:392 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-04 20:09:35,391 INFO    MainThread:392 [wandb_init.py:_log_setup():342] Logging user logs to D:\Users\Zumo\Documents\UNIVERSITÀ\AI\DL\Flatland_DDDQN\wandb\run-20210904_200935-z6ziqb72\logs\debug.log
2021-09-04 20:09:35,391 INFO    MainThread:392 [wandb_init.py:_log_setup():343] Logging internal logs to D:\Users\Zumo\Documents\UNIVERSITÀ\AI\DL\Flatland_DDDQN\wandb\run-20210904_200935-z6ziqb72\logs\debug-internal.log
2021-09-04 20:09:35,391 INFO    MainThread:392 [wandb_init.py:init():375] calling init triggers
2021-09-04 20:09:35,391 INFO    MainThread:392 [wandb_init.py:init():382] wandb.init called with sweep_config: {'gamma': 0.9179358796624579, 'hidden_size': 256, 'learning_rate': 0.006388775927541345, 'tau': 0.0038181419588845223}
config: {'n_episodes': 50, 'env_config': 0, 'n_evaluation_episodes': 3, 'checkpoint_interval': 10, 'eps_start': 1.0, 'eps_end': 0.01, 'eps_decay': 0.99, 'buffer_size': 100000, 'buffer_min_size': 0, 'restore_replay_buffer': '', 'save_replay_buffer': False, 'batch_size': 128, 'gamma': 0.9179358796624579, 'tau': 0.0038181419588845223, 'learning_rate': 0.006388775927541345, 'hidden_size': 256, 'update_every': 8, 'render': False, 'training_id': '210904200933', 'n_agents': 2, 'x_dim': 35, 'y_dim': 35, 'n_cities': 2, 'max_rails_between_cities': 2, 'max_rails_in_city': 3, 'malfunction_rate': 0.02, 'seed': 0, 'observation_tree_depth': 2, 'observation_radius': 10, 'observation_max_path_depth': 30}
2021-09-04 20:09:35,391 INFO    MainThread:392 [wandb_init.py:init():424] starting backend
2021-09-04 20:09:35,391 INFO    MainThread:392 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=spawn, using: spawn
2021-09-04 20:09:35,398 INFO    MainThread:392 [backend.py:ensure_launched():135] starting backend process...
2021-09-04 20:09:35,485 INFO    MainThread:392 [backend.py:ensure_launched():140] started backend process with pid: 18064
2021-09-04 20:09:35,486 INFO    MainThread:392 [wandb_init.py:init():429] backend started and connected
2021-09-04 20:09:35,487 INFO    MainThread:392 [wandb_run.py:_config_callback():843] config_cb None None {'gamma': 0.9179358796624579, 'hidden_size': 256, 'learning_rate': 0.006388775927541345, 'tau': 0.0038181419588845223}
2021-09-04 20:09:35,488 INFO    MainThread:392 [wandb_init.py:init():477] updated telemetry
2021-09-04 20:09:35,489 INFO    MainThread:392 [wandb_init.py:init():500] communicating current version
2021-09-04 20:09:37,253 INFO    MainThread:392 [wandb_init.py:init():505] got version response 
2021-09-04 20:09:37,253 INFO    MainThread:392 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-09-04 20:09:37,643 INFO    MainThread:392 [wandb_init.py:init():540] starting run threads in backend
2021-09-04 20:09:39,979 INFO    MainThread:392 [wandb_run.py:_console_start():1601] atexit reg
2021-09-04 20:09:39,981 INFO    MainThread:392 [wandb_run.py:_redirect():1475] redirect: SettingsConsole.WRAP
2021-09-04 20:09:39,982 INFO    MainThread:392 [wandb_run.py:_redirect():1512] Wrapping output streams.
2021-09-04 20:09:39,985 INFO    MainThread:392 [wandb_run.py:_redirect():1536] Redirects installed.
2021-09-04 20:09:39,985 INFO    MainThread:392 [wandb_init.py:init():565] run started, returning control to user process
2021-09-04 20:20:56,472 INFO    MainThread:392 [wandb_run.py:_atexit_cleanup():1571] got exitcode: 0
2021-09-04 20:20:56,472 INFO    MainThread:392 [wandb_run.py:_restore():1543] restore
2021-09-04 20:20:57,341 INFO    MainThread:392 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1005
  total_bytes: 1005
}

2021-09-04 20:20:57,686 INFO    MainThread:392 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1005
  total_bytes: 1005
}

2021-09-04 20:20:58,722 INFO    MainThread:392 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1005
  total_bytes: 8430
}

2021-09-04 20:20:58,832 INFO    MainThread:392 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1005
  total_bytes: 8430
}

2021-09-04 20:20:58,943 INFO    MainThread:392 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1005
  total_bytes: 8430
}

2021-09-04 20:20:59,053 INFO    MainThread:392 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8430
  total_bytes: 8430
}

2021-09-04 20:20:59,164 INFO    MainThread:392 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8430
  total_bytes: 8430
}

2021-09-04 20:20:59,273 INFO    MainThread:392 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8430
  total_bytes: 8430
}

2021-09-04 20:20:59,381 INFO    MainThread:392 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8430
  total_bytes: 8430
}

2021-09-04 20:20:59,488 INFO    MainThread:392 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8430
  total_bytes: 8430
}

2021-09-04 20:20:59,598 INFO    MainThread:392 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8430
  total_bytes: 8430
}

2021-09-04 20:20:59,965 INFO    MainThread:392 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8430
  total_bytes: 8430
}

2021-09-04 20:21:00,080 INFO    MainThread:392 [wandb_run.py:_wait_for_finish():1693] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8430
  total_bytes: 8430
}

2021-09-04 20:21:01,369 INFO    MainThread:392 [wandb_run.py:_show_summary():1848] rendering summary
2021-09-04 20:21:01,374 INFO    MainThread:392 [wandb_run.py:_show_history():1886] rendering history
2021-09-04 20:21:01,381 INFO    MainThread:392 [wandb_run.py:_show_files():1915] logging synced files
